build.gradle

```diff
@@ -112,7 +112,9 @@ dependencies {
   compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
   compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
 
-  compile 'org.hibernate:hibernate-java8:5.0.12.Final'
+  compile 'org.hibernate:hibernate-java8:5.2.11.Final'
+  compile 'org.hibernate:hibernate-core:5.2.11.Final'
+  compile 'org.hibernate:hibernate-entitymanager:5.2.11.Final'
```

Comment:
for POC that's fine, for the pilot we have to check if we can also change version whenever needed.

Reply:
🥇